app/components/datepicker/datepicker.less

```diff
@@ -42,3 +42,155 @@
 .DatePicker-control--year {
   width: 80px;
 }
+
+/* Popup calendar */
+/* Parent element must have: position: relative; */
+.datepicker-popup {
+  position: absolute;
+  top: 2em;
+  z-index: 99;
+  display: flex;
+  flex-direction: column;
+  padding: 0.5em;
+  background-color: white;
+  color: @black-color;
+  border-width: thin;
+  border-style: solid;
+  border-color: #ccc;
+  cursor: initial;
+}
+
+.datepicker-popup-head {
+  display: flex;
+  justify-content: space-between;
+  color: @gray-title;
+  font-weight: bold;
+  cursor: default;
+}
+
+.datepicker-popup-year {
+  color: #281946;
+}
+
+.datepicker-popup-monthdays {
+  display: grid;
+  grid-template-columns: repeat(7, 2rem);
+  margin-top: 0.4rem;
+  margin-bottom: 0.2rem;
+}
+
+.datepicker-popup-weekday {
+  color: #727375;
+  font-weight: bold;
+  cursor: default;
+  margin-top: 0.2rem;
+  margin-bottom: 0.2rem;
+}
+
+.datepicker-popup-change-month {
+  color: @black-color;
+  cursor: pointer;
+  margin-left: 0.2rem;
+  margin-right: 0.2rem;
+
+  &:hover {
+    color: @gray-dark;
+    text-decoration-line: underline;
+    font-weight: bold;
+  }
+}
+
+.datepicker-popup-day {
+  cursor: pointer;
+  border-style: solid;
+  border-color: transparent;
+  border-radius: 40%;
+  border-width: 0.2rem;
+  margin-top: 0.2rem;
+  margin-bottom: 0.2rem;
+  color: #281946;
+
+  &:hover {
+    color: @gray-dark;
+    text-decoration: underline;
+    font-weight: bold;
+  }
+}
+
+.datepicker-popup-day-disabled {
+  background-color: @purple-light;
+  cursor: default;
+}
+
+.datepicker-popup-day-out {
+  font-weight: lighter;
+  color: #ccc;
+}
+
+.datepicker-popup-day-selected {
+  font-weight: bold;
+  color: @white-color;
+  background-color: @gray-dark;
+  border-color: @gray-dark;
+
+  &:hover {
+    color: @white-color !important;
+    font-weight: normal !important;
+  }
+}
+
+/* Range calendar */
+.datepicker-popup-range {
```

Comment:
this one may cause issue like that 
![image](https://user-images.githubusercontent.com/22294614/73369907-ab600f80-42b3-11ea-8a14-fe8c265d0200.png)

Reply:
Yes, I need to improve the responsive design.